softwerkskammer/lib/mailarchive/importMails.js

```diff
@@ -31,14 +31,12 @@ module.exports = function (file, group, done) {
       return;
     }
 
-    var shasum = crypto.createHash('sha1');
-    var s = fs.ReadStream(file);
-    s.on('data', function (d) {
-      shasum.update(d);
-    });
+    const shasum = crypto.createHash('sha1');
+    const s = fs.ReadStream(file);
```

Comment:
Sollte man hier auch mal einen Umbau auf asynchron in Erwägung ziehen, so wie ich das bei dem git-Modul gemacht hab? (oder ist das hier was anderes?)